src/threadlist.cpp

```diff
@@ -572,6 +572,13 @@ stopthisthread(int signum)
    * sigaction(STOPSIGNAL, stopthisthread) to discard all pending signals.
    */
 
+  // If the user sent a ckpt signal, it can be caught by either ckpt-thread or
```

Comment:
In above line, let's clean up the comment now;
1.  Case 3 above: insert newline before it
2. Case 4 above:  ckpt-threads -> ckpt-thread

The line just above here, is not clear.  "If the user sent a ckpt signal":  The "user" can send a command to the coordinator for a checkpoint.  The "user" should not directly send a ckpt signal to some random thread.  Does this comment mean that we are in Case 4 here.  If so, please just say that we're in Case 4, above.